DataFormats/OnlineMetaData/interface/OnlineMetaDataRaw.h

```diff
@@ -0,0 +1,67 @@
+#ifndef DATAFORMATS_ONLINEMETADATA_ONLINEMETADATARAW_H
+#define DATAFORMATS_ONLINEMETADATA_ONLINEMETADATARAW_H
+
+//---------------------------------------------------------------------------
+//!  \class OnlineMetaDataRaw
+//!  \brief Structure of raw data from soft FED 1022
+//!
+//!  \author Remi Mommsen - Fermilab
+//---------------------------------------------------------------------------
+
+#include <cstdint>
+#include "EventFilter/FEDInterface/interface/fed_header.h"
+#include "EventFilter/FEDInterface/interface/fed_trailer.h"
```

Comment:
this is not a legal dependence 
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Restrictions_on_the_package_defi

this can be better resolved by moving fed_header and trailer to DataFormats, which should also resolve #21312

Reply:
I think this change would warrant a different pull request, as it will affect quite a few packages. I'll look into it.

Reply:
it can be a different pull request, but in that case this PR will need to wait.

Reply:
#21690 is now merged and changes related to it can be now completed